lib/java_buildpack/framework/jrebel_agent.rb

```diff
@@ -24,36 +24,51 @@ module Framework
     # Encapsulates the functionality for enabling zero-touch JRebel support.
     class JrebelAgent < JavaBuildpack::Component::VersionedDependencyComponent
 
+      def initialize(context, &version_validator)
+        super(context, &version_validator)
+        @component_name = 'JRebel Agent'
+      end
+
       # (see JavaBuildpack::Component::BaseComponent#compile)
       def compile
-        download_zip false
-        FileUtils.mv(download_location + 'jrebel.jar', @droplet.sandbox + jar_name)
-        FileUtils.remove_dir(download_location, true)
+        download_zip(true, @droplet.sandbox + 'install')
       end
 
       # (see JavaBuildpack::Component::BaseComponent#release)
       def release
         @droplet.java_opts
-          .add_javaagent(@droplet.sandbox + jar_name)
-          .add_bootclasspath_p(@droplet.sandbox + jar_name)
+          .add_agentpath(@droplet.sandbox + ('install/lib/' + lib_name))
           .add_system_property('rebel.remoting_plugin', true)
+          .add_system_property('rebel.log', true)
+          .add_system_property('rebel.base', @droplet.sandbox + '.jrebel')
```

Comment:
The existence of the sandbox beans that you don't need to put anything in a `.jrebel` directory, you're already in one.  `rebel.base` should probably be `@droplet.sandbox` only.